### ISSUE TITLE
Feature: When creating a new meeting via MeetingBar, copy the join URL to the clipboard

### ISSUE THREAD

**Issue by roger-o3h** [leits/MeetingBar#198]
**Is your feature request related to a problem? Please describe.**
Currently to invite someone to an ad-hoc meeting, I'll use MeetingBar to create a meeting in my preferred service with one click. Then it's another few keystrokes and/or clicks to fetch the link to the ad-hoc meeting to share it.

**Describe the solution you'd like**
MeetingBar creates and opens the meeting when you click Create Meeting, and it puts the URL to the newly-created meeting in my clipboard. Now inviting someone to a new meeting would take 2 keystrokes... amazing!

**Describe alternatives you've considered**
I thought maybe this could be achieved with "Run applescript when joining to meeting". Also I could copy the URL manually like a not-lazy person :)

**Additional context**
Thanks again for all your work on this awesome utility, and for considering feature requests!

**Comment by roger-o3h** [comment]
It occurs to me that if both this and feature request #197 are implemented we're looking at a potential circular infinite loop, haha. Please consider both low priority suggestions :)

**Comment by roger-o3h** [comment]
This seems to do the trick for now as a custom join script:

```
tell application "Google Chrome"
	set var to URL of active tab of front window
	set the clipboard to var
end tell
```

**Comment by leits** [comment]
BLUF: Sorry, but it's impossible to make it into the app at all.

To open the link, the app calls the macOS method, which returns only a true/false.
Therefore, the application knows absolutely nothing about the browser. This is a by-design in macOS for application isolation.
But you manage to solve it with the help of AppleScript.

The next nuance is that some of the links lead to the form of creating a meeting and only after filling it out the user will receive a link to the meeting itself. Some links lead to a page with a redirect, and then you get to the meeting page.

Sounds like it makes sense to run the script when attached to a meeting. But only for those links that make a redirect and get to the meeting itself. And this is very implicit behavior.
